Gopkg.toml

```diff
@@ -1,5 +1,6 @@
 required = [
   "github.com/golang/protobuf/protoc-gen-go",
+  "github.com/gordonklaus/ineffassign",
```

Comment:
I see that this project doesn't do releases, but mind pinning it to a fixed sha in this file?